Makefile

```
@@ -152,15 +152,15 @@ scripts/vendor: jb scripts/jsonnetfile.json scripts/jsonnetfile.lock.json
 	cd scripts && jb install
 
 jsonnet:
-	GO111MODULE=off go get github.com/google/go-jsonnet/cmd/jsonnet
+	GO111MODULE=on go install github.com/google/go-jsonnet/cmd/jsonnet
```

Since we are using go 1.13, we don't need to set the `GO111MODULE` env var.
```suggestion
	go install github.com/google/go-jsonnet/cmd/jsonnet
```

---

Alright I'll remove them